engine/utils/Macros.h

```diff
@@ -24,4 +24,8 @@
 #define STRINGIFY(x) #x
 #define TOSTRING(x) STRINGIFY(x)
 
+// Macro for declaring a binary operator
+#define DECL_BINARY_OP_NO_IMP(op, returnType, lhsType, rhsType) \
```

Comment:
For future: `DECL_BINARY_OP_NO_IMPL`

Reply:
I don''t understand this comment. Not sure what you're trying to say here.